### ISSUE TITLE
Feature Request: Hover over palette event

### ISSUE THREAD

**Issue by legojoe** [bgrins/spectrum#288]
It would be perfect to have an event fired when hovering over the palette. Or perhaps extend the 'move' event to allow this?

**Comment by bgrins** [comment]
What is the use case for this event?  Do you want to change the color when hovering a palette item?

**Comment by legojoe** [comment]
Yes, exactly  - to give the user a preview before they commit to the change. Not essential, but a really really really nice to have :)
